<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartMenu</class>
 <widget class="QWidget" name="StartMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="BtnBasic">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>99</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">image: none;
background-image:none;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>基本模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnZen">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>99</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">image: none;
background-image:none;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>休闲模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnLevel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>99</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">image: none;
background-image:none;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>关卡模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnOptions">
   <property name="geometry">
    <rect>
     <x>644</x>
     <y>540</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: none;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnHelp">
   <property name="geometry">
    <rect>
     <x>728</x>
     <y>540</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: none;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnRank">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>540</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: none;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>排行榜</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
